src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -180,11 +173,9 @@ const DiscoverySearch = ({
       </View>
       <View style={styles.searchContainer}>
         <View style={styles.wrapperSearch}>
-          {isSearchIconShown && (
-            <View style={styles.wrapperIcon}>
-              <MemoIcSearch width={16.67} height={16.67} />
-            </View>
-          )}
+          <View style={styles.wrapperIcon}>
+            <MemoIcSearch width={16.67} height={16.67} />
+          </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `isSearchIconShown` has been removed, which means the search icon will always be shown. If this change was intentional and doesn't affect the UI/UX negatively, then it's fine.
```diff
- {isSearchIconShown && (
  <View style={styles.wrapperIcon}>
    <MemoIcSearch width={16.67} height={16.67} />
  </View>
- )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->